Automattic/pocket-casts-android
# [Compose Migration] Migrate shelf list items to Compose - Part i

## Description

This 
- Migrates shelf list row items (from more option in the Player shelf) to Compose
- Removes bookmark item highlight on what's new (bookmark is not a new feature anymore and the shelf item doesn't need to be highlighted)

## Testing Instructions
1. Play an episode
2. Open full-screen player
3. Tap more option in the player shelf
4. ✅ Notice that `More actions` shelf items are properly shown and you can click on them as before
5. Tap edit action
6. ✅ Notice that `ReArrange Actions` shelf items are properly shown and you can rearrange them as before

## Screenshots or Screencast 

Before

More Actions | Rearrange Actions
---|----
<img width=200 src="https://github.com/user-attachments/assets/f8f87ccb-6269-4a42-8a32-dbef3e3b9a41"/>|<img width=200 src="https://github.com/user-attachments/assets/62f19670-d66a-47e3-acad-ceaddafc6c05"/>

After

More Actions | Rearrange Actions
---|----
<img width=200 src="https://github.com/user-attachments/assets/6e29682c-b5b7-4e17-80a9-0d9425c1d1cf"/>| <img  width=200 src="https://github.com/user-attachments/assets/507bd0a8-1be8-4ed7-a9b7-05496cdc1f6b"/>


https://github.com/user-attachments/assets/0363da67-0e24-4163-b353-5290e551ac66


## Checklist
 
#### I have tested any UI changes...
